jjb/edgex-templates-snap.yaml

```diff
@@ -368,3 +368,37 @@
       - lf-provide-maven-settings-cleanup
       - edgex-provide-snap-cleanup
       - edgex-provide-docker-cleanup
+- job-template:
+    name: '{project-name}-release-snap'
+
+    # Job template for Snap merge jobs
+    #
+    # The purpose of this job template is to run snap build
```

Comment:
Isn't the purpose of this job template to run` snapcraft release`?

Reply:
copy & paste issue again I'll change this